src/stopwatch.ts

```diff
@@ -26,14 +26,35 @@ export class StopWatch {
 
   // TODO ⏲
   private init() {
-    this.events$.pipe(tap((e) => console.log(e))).subscribe();
+    this.events$
+      .pipe(
+        startWith(initalState),
+        scan(
+          (state: State, payload: Partial<State>) => ({
+            ...state,
+            ...payload
+          }),
+          initalState

```

Comment:
ez hianyzott a workshopnal, nem adtuk meg a scan()-nek hogy milyen ertekkel induljon, igy a startWith nem is kell